test/test_bundle.js

```diff
@@ -767,10 +767,6 @@ describe('json-pollock tests', function () {
       chai.expect(carouselRight.className).to.contain('lp-json-pollock-layout-carousel-arrow');
     });
 
-    it('carousel arrow right has role attribute', function () {
```

Comment:
replace it with validating the `type` attribute

Reply:
Great point, thanks! Done